Card/src/main/java/com/paypal/android/card/CardRequest.kt

```diff
@@ -1,9 +1,17 @@
 package com.paypal.android.card
 
-import com.paypal.android.card.threedsecure.ThreeDSecureRequest
+import com.paypal.android.card.threedsecure.SCA
 
-data class CardRequest(
+/**
+ * A card request to process a card payment
+ * @property orderID The order ID to to process the payment
+ * @property card   Card used for payment
+ * @property returnUrl Url to return to app after SCA challenge finishes
+ * @property sca Specify to always launch 3DS or only when required. Defaults to `SCA.SCA_WHEN_REQUIRED`.
+ */
+data class CardRequest @JvmOverloads constructor(
     val orderID: String,
     val card: Card,
-    var threeDSecureRequest: ThreeDSecureRequest
+    val returnUrl: String,
+    val sca: SCA = SCA.SCA_WHEN_REQUIRED
 )
```

Comment:
I like this change! Now that 3DS is non-optional, and there was only 2 params on it, this is more explicit.